### DISCUSSION TITLE
SOP: Priority Field

### DISCUSSION THREAD

**Discussion by dan-menlo** [janhq/jan#3596] [How We Work]
## Overview

- Priority is a Github Projects column that is mainly used by PMs
- Used to triage the Product Backlog (i.e. Urgent + Important tasks are at the top)

**Comment by LazyYuuki** [comment]
Current SOP for Priority field labeling for each issue type:

## Bug
1. `P-Critical` = break core function, urgent, fix immediately / in next release
2. `P-High` = bug that break normal user flow but not core function,
3. `P-Medium` = unexpected behavior but does not break normal user flow,
4. `P-Low` = minor annoyance but does not break normal user flow

## Feature / Enhancement
1. `P-critical`: urgently need this feature to prevent user from encountering breaking workflow => release immediately / in next release
2. `P-high`:  actively be worked on and will be in the next 1 or 2 release (patch for enhancement / minor for feature)
3. `P-medium`:  will be worked on next after P-high
4. `P-low`: will only be worked on if we are low on issue and it improves UX - can also be delegated to community for contribution (we have not tapped onto this source yet)

##Epic
1. There shouldn't be `P-critical` epic, epic is meant to be well planned (exception: epic = Jan is Unicorn 🦄 )
2. `P-high`: actively be worked on and will be finished in the next 1 or 2 Minor bump
3. `P-medium`: follow after `P-high`
4. There shouldn't be `P-low` epic, that just meant it is a feature or it is not important to drive Jan's usage (a.k.a we shouldn't focus resource on it)
